core/templates/dev/head/pages/exploration_player/correctness_footer_directive.html

```diff
@@ -0,0 +1,60 @@
+<style>
+  .inner-container {
+    background: rgb(221, 248, 233);
+    border:1px;
```

Comment:
For all these:

- add single space after colon
- alphabetize attrs in each rule
- prefix all CSS classes with `correctness-footer`, e.g. `correctness-footer .inner-container`, so that they apply only to the correctness footer and nothing else